app/views/events/index.html.erb

```diff
@@ -2,49 +2,36 @@
 
 <h1>Events</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Title</th>
-      <th>Description</th>
-      <th>Date</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @events.each do |event| %>
-      <tr>
-        <td><%= event.title %></td>
-        <td><%= event.description %></td>
-        <td><%= event.date %></td>
-        <td><%= link_to 'Show', event %></td>
-        <td><%= link_to 'Edit', edit_event_path(event) %></td>
-        <td><%= link_to 'Destroy', event, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
 
 <br>
-<h2>Upcoming Events</h2>
-<% if @events.upcoming.any? %>
-	<ul class="media-list">
-		<%= render @events.upcoming %>
-	</ul>
 
-<% else %>
-	<h3>Events (0)</h3>
-	<%= link_to new_event_path, class: 'btn btn-success' do %>
-		Add Event <i class="fa fa-plus-circle"></i>
-	<% end %>
+<div class = "container">
+<h2>Upcoming events</h2>
+<div class = "events">
+<% @events.upcoming.each do |event|%>

```

Comment:
This could be resolved in the controller and the variable @upcoming_events passed to the view.